graylog2-web-interface/src/components/common/SortableListItem.jsx

```diff
@@ -24,7 +24,7 @@ import { ListGroupItem } from 'components/graylog';
 import Icon from './Icon';
 
 // eslint-disable-next-line import/no-webpack-loader-syntax
-import SortableListItemStyle from '!style!css!components/common/SortableListItem.css';
+import SortableListItemStyle from '!style-loader!css-loader!components/common/SortableListItem.css';
```

Comment:
Are these prefixes needed? Maybe a good time to try and import without them.

Reply:
Yes, they are required now. Starting with webpack 5, the shortcut has been removed. IMO it is not a big deal for us, as we would use styled-components for new code anyway and slowly move existing code over if needed.

Reply:
Alright! Then let's move away from it later.